internal/cmdliveinit/cmdliveinit.go

```diff
@@ -149,22 +153,13 @@ func (c *ConfigureInventoryInfo) Run(ctx context.Context) error {
 	} else {
 		name = c.Name
 	}
-	// Generate the inventory id if one is not specified through a flag.
-	if c.InventoryID == "" {
-		id, err := generateID(namespace, name, time.Now())
-		if err != nil {
-			return errors.E(op, c.Pkg.UniquePath, err)
-		}
-		inventoryID = id
-	} else {
-		inventoryID = c.InventoryID
-	}
+	inventoryID = c.InventoryID
 	// Finally, update these values in the Inventory section of the Kptfile.
```

Comment:
we also want to check the Kptfile and make sure the inventory does not exist. otherwise warn users

Reply:
Done.